server/Procfile

```diff
@@ -1,4 +1,4 @@
-web: opentelemetry-instrument gunicorn --bind 0.0.0.0:$PORT --workers 1 avocano_api.wsgi:application
+web: gunicorn --bind 0.0.0.0:$PORT --workers 1 avocano_api.wsgi:application
```

Comment:
In moving from auto-instrumentation to inline, it feels like we need to capture some of the context somewhere. Could we add some documentation about instrumentation approach/decisions to the PR?

Reply:
Added as a doc, this repo's version of ADR